.travis.yml

```diff
@@ -14,4 +14,4 @@ script:
 after_script:
   - make docs_report
 after_success:
-  - make push_docs_report
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
@milmazz can't be on the `Makefile`?

Reply:
@yordis At least as is no, is a syntax error.